src/v/config/configuration.cc

```diff
@@ -1526,7 +1526,13 @@ configuration::configuration()
       "Maximum capacity of rate limit accumulation"
       "in controller configuration operations limit",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
-      std::nullopt) {}
+      std::nullopt)
+  , health_monitor_append_entries_timeout_ms(
```

Comment:
We might change the mechanism for this in future (to use something other than append entries messages as the signal for isolation), so let's make the configuration property name more generic.

Something like "node_isolation_timeout" perhaps?